CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## [1.2.9](https://github.com/ably/ably-cocoa/tree/1.2.9)
+
+[Full Changelog](https://github.com/ably/ably-cocoa/compare/1.2.8...1.2.9)
+
+**Bug Fix:**
+
+This bug affects customers using
+[message content encryption](https://ably.com/documentation/realtime/encryption)
+alongisde
```

Comment:
I know that message encryption was working fine in other cases (publishing, subscribing). I wonder if we should add this bit 🤔

Reply:
I'm not quite sure what you're driving at... I've used the word 'alongside' to emphasis that this is a cumulative effect of using several features together. The context of this change log is that app developers are likely to want to assess quickly whether this bug fix will affect their use case or not, hence this introductory paragraph.